as-pipeline-sim/functions/functions.go

```diff
@@ -286,66 +288,183 @@ func (p *PipelineSim) TriggerGetiPipeline(ctx interfaces.AppFunctionContext, dat
+	payload2 := []byte(fmt.Sprintf(`{
+		"source": {
+			"path": "%s",
+			"type": "file"
+		}
+	}`, p.params.InputFileLocation))
+
+	p.lc.Debugf("Inference call to EVAM...")
+	// Inference call
+	_, err = executePostRequest(url2, payload2)
 	if err != nil {
-		err = werrors.WrapMsg(err, "failed to create http request to trigger Geti pipeline.")
+		err = werrors.WrapMsg(err, "failed to trigger inference call EVAM pipeline request.")
 		p.lc.Errorf("TriggerGetiPipeline failed: %s", err.Error())
 		return true, err
 	}
 
-	response, err := http.DefaultClient.Do(request)
-	if err != nil {
-		err = werrors.WrapMsg(err, "failed to trigger Geti pipeline request.")
+	p.lc.Debugf("Deleting EVAM pipeline...")
+	deleteUrl := fmt.Sprintf("%s/pipelines/%s", p.config.GetiUrl, instanceID)
+	// Delete pipeline
+	if err := deleteInstance(deleteUrl); err != nil {
+		err = werrors.WrapMsg(err, "failed to delete EVAM pipeline.")
 		p.lc.Errorf("TriggerGetiPipeline failed: %s", err.Error())
 		return true, err
 	}
 
-	if response.StatusCode != http.StatusOK {
-		err := fmt.Errorf("trigger Geti pipeline request failed with status code %d", response.StatusCode)
+	p.lc.Debugf("Move output file to its destination...")
+	// Move file to the correct destination
+	if err := moveFile("/tmp/files/output/output.jpg", outputFilenamePath); err != nil {
 		p.lc.Errorf("TriggerGetiPipeline failed: %s", err.Error())
 		return true, err
 	}
 
-	resbody, err := io.ReadAll(response.Body)
+	p.lc.Debugf("Read inference data from jsonl file...")
+	// Read inference output from jsonl file
+	outputFile := fmt.Sprintf("/tmp/%s.jsonl", filename)
+	output, err := readJSONLFile(outputFile)
 	if err != nil {
-		err := werrors.WrapMsg(err, "failed to read response from Geti pipeline")
 		p.lc.Errorf("TriggerGetiPipeline failed: %s", err.Error())
 		return true, err
 	}
 
-	var pipelineResp pipelineBodyResp
-	err = json.Unmarshal(resbody, &pipelineResp)
+	p.pipelineResults = output
+
+	return true, data
+}
+
+func readJSONLFile(filename string) (string, error) {
+	file, err := os.Open(filename)
 	if err != nil {
-		err = werrors.WrapMsgf(err, "unable to unmarshal data from Geti response")
-		p.lc.Errorf("TriggerGetiPipeline failed: %s", err.Error())
-		return true, err
+		err = werrors.WrapMsg(err, "failed to open file")
+		return "", err
+	}
+	defer file.Close()
+
+	var lines []string
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
 	}
 
-	output, err := json.Marshal(pipelineResp)
+	if err := scanner.Err(); err != nil {
+		err = werrors.WrapMsg(err, "failed to read file")
+		return "", err
+	}
+
+	//TODO: this throws operation not permitted. Need to investigate a solution
+	// Delete the file after reading
+	// if err := os.Remove(filename); err != nil {
+	// 	err = werrors.WrapMsg(err, "failed to delete file")
+	// 	return "", err
+	// }
+
+	return strings.Join(lines, "\n"), nil
```

Comment:
use platform independent newline:
```suggestion
	return strings.Join(lines, fmt.Sprintln()), nil
```